components/automate-ui/src/app/entities/layout/layout.actions.ts

```diff
@@ -1,22 +1,29 @@
 import { Action } from '@ngrx/store';
-import { MenuItemGroup } from './layout.model';
+import { Sidebars } from './layout.model';
 
 export enum LayoutActionTypes {
   SHOW_PAGE_LOADING = 'Layout::SHOW_PAGE_LOADING',
-  UPDATE_SIDEBAR_MENU_GROUPS = 'LAYOUT::UPDATE'
+  GET_SIDEBAR = 'LAYOUT::GET_SIDEBAR',
+  UPDATE_SIDEBARS = 'LAYOUT::UPDATE_SIDEBARS'
 }
 
 export class ShowPageLoading implements Action {
   readonly type = LayoutActionTypes.SHOW_PAGE_LOADING;
   constructor(public payload: boolean) {}
 }
 
-export class UpdateSidebarMenuGroups implements Action {
-  readonly type = LayoutActionTypes.UPDATE_SIDEBAR_MENU_GROUPS;
+export class GetSidebar implements Action {
```

Comment:
Again, I'm new here-- is there one sidebar or many? We have a `GetSidebar` but `UpdateSidebars` in the plural

Reply:
@blakestier we have sidebars and we get one of those sidebars.